packages/cmf/src/middlewares/http/middleware.js

```diff
@@ -12,7 +12,7 @@ export const DEFAULT_HTTP_HEADERS = {
 };
 
 export function isHTTPRequest(action) {
-	return action.type in HTTP_METHODS;
+	return action && action.type in HTTP_METHODS;
```

Comment:
If the user provide a custom method on the onError and this method doesn't provide a type, this trigger errors on CMF. I can add a check on this.